src/Microsoft.DocAsCode.Build.SchemaDriven/SchemaExtensions.cs

```diff
@@ -3,22 +3,24 @@
 
 namespace Microsoft.DocAsCode.Build.SchemaDriven
 {
+    using System;
+    using System.Linq;
+
     public static class SchemaExtensions
     {
-        private const string IsRequiredInFragmentsTag = "editable";
-
-        private const string IsLegalInFragmentsTag = "editable";
+        private const string IsEditableTag = "editable";
```

Comment:
>private const string IsEditableTag = "editable"; [](start = 8, length = 48)

tends to use single "editable" const, but keep 2 methods in case of future change